src/components/layout/Navbar/index.tsx

```diff
@@ -50,6 +81,34 @@ const Navbar = ({ user }: NavbarProps) => {
           </Link>
         </nav>
       </div>
+      {/* Mobile Menu Dropdown */}
+      <div className={`${styles.mobileNav} ${menuOpen && styles.open}`}>
+        <Link className={styles.mobileNavItem} onClick={() => setMenuOpen(false)} href="/">
```

Comment:
```suggestion
        <Link className={styles.mobileNavItem} onClick={() => setMenuOpen(false)} href={config.homeRoute} >
```
Let's try to avoid magic variables for important routes (home, login, admin) to keep code readable as to what it's doing at a high level and make modifications easy if we ever needed to move the page route.